shared/build.gradle.kts

```
@@ -24,8 +24,13 @@ kmmbridge {
 }
 
 sqldelight {
+    // Read more on gradle options on https://cashapp.github.io/sqldelight/multiplatform_sqlite/gradle/
     database("CioDatabase") {
         packageName = "io.customer.shared.local"
+        // The directory where to store '.db' schema files to verify migrations
+        schemaOutputDirectory = file("src/main/sqldelight/databases")
```

To try and limit the number of directories in this project, can we put the schema files in an existing location? 

```suggestion
        schemaOutputDirectory = file("src/commonMain/sqldelight/schema")
```